sway-core/src/semantic_analysis/namespace/module.rs

```diff
@@ -52,110 +51,97 @@ pub struct Module {
 }
 
 impl Module {
-    pub fn default_with_constants(
+    pub fn default_with_contract_id(
```

Comment:
Let's also add a similar comment here, otherwise I imagine this might confuse compiler-devs quite a lot, as sway-core really shouldn't know anything about contract IDs :sweat_smile:! We might also want to add a link to the issue which states how we plan to refactor this out of sway-core in favour of creating temporary package dependencies for providing these `CONTRACT_ID`-containing modules.